src/pages/glossary.js

```diff
@@ -24,7 +24,7 @@ export default ({ data }) => {
 							<article key={node.id}>
 								<section>
 									<a id={key} href={`#${key}`}>
-										<h2>{frontmatter.title}</h2>
+										<h3>{frontmatter.title}</h3>
```

Comment:
Don't skip heading levels. Do something else if you need the styles changes on these.